script.js

```diff
@@ -204,3 +204,39 @@ form.addEventListener('submit', (event) => {
     form.submit();
   }
 });
+
+const fullname = document.querySelector('#fullname');
+const message = document.querySelector('#message');
+
+const inputData = {};
+const storage = window.localStorage;
+
+fullname.addEventListener('change', () => {
+  inputData.fullname = fullname.value;
+  inputData.email = email.value;
+  inputData.message = message.value;
+  storage.setItem('formData', JSON.stringify(inputData));
+});
+
+email.addEventListener('change', () => {
+  inputData.fullname = fullname.value;
+  inputData.email = email.value;
+  inputData.message = message.value;
+  storage.setItem('formData', JSON.stringify(inputData));
+});
+
+message.addEventListener('change', () => {
+  inputData.fullname = fullname.value;
+  inputData.email = email.value;
+  inputData.message = message.value;
+  storage.setItem('formData', JSON.stringify(inputData));
+});
```

Comment:
- [ ] Great job on implementing the project requirement! 👏 However, there is a minor issue that we need to address to make your code follow best practices and also make it look professional. You can see for yourself that you're repeating the same pieces of code several times 😊 which isn't the best. What if we can refactor this code to make it `DRY(Don't Repeat Yourself)` and improve its readability? Consider below suggestions to do this:
  - [ ] Create a function, such as `updateFormData`, that contains the shared logic for updating the `inputData` object and storing it in `localStorage`. Example 👇
  ```
       const updateFormData = () => {
             const inputData = {
                // Include logic for updating user input
            }
            // Save inputData to localStorage
        }
   ```
   
     - [ ] Modify the event listeners: Instead of duplicating the code in each event listener, you should consider attaching the `updateFormData` function as the event listener for the `change` event on the `fullname`, `email`, and `message` elements. Example 👇
   
   ``` 
     fullname.addEventListener('change', updateFormData);
   ```

Reply:
Thank you, me and my partner will start working on it

Reply:
You're welcome😊 @SnakyMz. In case you don't understand anything please don't hesitate to ask.